.release-please-manifest.json

```
@@ -8,5 +8,6 @@
   "libs/providers/launchdarkly-client": "0.2.1",
   "libs/providers/go-feature-flag-web": "0.1.6",
   "libs/shared/flagd-core": "0.1.11",
+  "libs/shared/ofrep-core": "0.0.1",
```

Can we start with an experimental version here maybe?

```suggestion
  "libs/shared/ofrep-core": "0.0.1-experimental",
```